lib/table-head/table-head.test.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import TableHead from './table-head';
+
+import { mount } from 'enzyme';
+
+let wrapper;
+
+describe('<TableHead />', () => {
+  beforeEach(() => {
+    wrapper = mount(
+      <TableHead>
+        <tr />
+      </TableHead>
+    );
+  });
+
+  it('Handles props.children', () => {
+    expect(wrapper.html()).toBe('<thead class="oc-thead"><tr></tr></thead>');
+
+    wrapper.setProps({
+      children: (
+        <tr>
+          <td />
+        </tr>
+      )
+    });
+    expect(wrapper.html()).toBe(
+      '<thead class="oc-thead"><tr><td></td></tr></thead>'
+    );
+  });
+
+  it('Handles props.className', () => {
+    expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-thead');
```

Comment:
not a blocker but consider wrapping those repeated statements into a function like
getWrapperAttribute('class');
getWrapperAttribute('style');

Reply:
just for the tests I mean, a language/functions specific to your test

Reply:
Hi Victor,

I've removed the comments.

I understand what you are saying. I will create a reusable method globally to do this. I'll write up a support ticket separately, as I'll make this change globally.

Merging.

TY.